openmetadata-docs/ingestion/connectors/postgres/usage.yaml

```diff
@@ -0,0 +1,31 @@
+source:
+  type: postgres-usage
+  serviceName: "<service name>"
+  serviceConnection:
+    config:
+      type: Postgres
+      username: openmetadata_user
+      password: openmetadata_password
+      hostPort: localhost:5432
+      database: pagila
```

Comment:
Should not these be something like `<database username>` and etc? Looks like expected values.